gini/gini-vision-lib-android
# Feature multi page memory management

Loading documents' data and creating previews are handled by memory cache managers backed by an LRU cache. The size of the LRU cache is determined by the available memory (kudos to @EarlOfEgo and the Switch SDK team). The memory cache managers are called `DocumentDataMemoryCache` and `PhotoMemoryCache`.

Storing images on disk is handled by the `ImageDiskStore` class. This writes byte arrays to the app's internal folder and returns a`Uri` which is then used in documents to read their data from.

### How to test
Run the Screen API Example App and take/import a lot of images and see if the app crashes.

### Ticket 
Issue #148
